android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebChromeClient.java

```diff
@@ -87,6 +88,15 @@ public boolean onConsoleMessage(ConsoleMessage message)
 		return true;
 	}
 
+	@Override
+	public void onPermissionRequest(final PermissionRequest request)
+	{
+		TiViewProxy proxy = tiWebView.getProxy();
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			request.grant(request.getResources());
```

Comment:
@jquick-axway Should we always grant this? I feel this might cause side-effects for other components.

Reply:
You still have to show the popup in your app to access camera (camera permission)

Reply:
So what do we do if a customer wants to explicitly disallow it? :-)

Reply:
The he clicks `no` when you ask for `Ti.Android.requestPermissions()` in your app. But you are right: you can't allow the camera in other parts of your app and disallow it for the webview. 
So a property/method would be better to allow it and then the developer has to reload the page when it is set to true!